processor/schemaprocessor/metadata.yaml

```diff
@@ -0,0 +1,7 @@
+type: schema
+
+status:
+  class: processor
+  stability:
+    development: [traces, metrics, logs]
+  distributions: [sumo]
```

Comment:
I believe we want to add `contrib` to the distributions here.

Reply:
Oh, it's not listed here: https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/manifest.yaml do you mind filing an issue to get it added @crobert-1 ? Thanks!

Reply:
@MovieStoreGuy is this component ready to be added to contrib? I got the sense it was still very much under development

Reply:
This might be something of my own confusion. The code for this processor is in `contrib`, so I assumed that meant it should be listed as one of the distributions. Am I missing something here? Is this something where we don't include in the distribution (even if the code is in `contrib`) until it meets a certain bar?